servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -4695,6 +4695,25 @@ void processUserAuthorityRestrictions() {
         }
     }
 
+    Map<String, List<String>> getDomainNamesFromTemplate(Map<String, Integer> templateDetails) {
```

Comment:
the name of the method does not reflect what the method is doing. looks like we're actually doing the auto update there and not just getting domain names from the template

Reply:
Done changed it.